elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/AggregationDataStore.java

```diff
@@ -63,6 +65,9 @@ public void populateEntityDictionary(EntityDictionary dictionary) {
 
     @Override
     public DataStoreTransaction beginTransaction() {
-        return new AggregationDataStoreTransaction(queryEngine, cache, queryLogger);
+        if (queryEngine.getClass().equals(SQLQueryEngine.class)) {
```

Comment:
Why are we doing this here?  I don't think we want AggregationDataStore to know about its subclasses.

Reply:
reverted.